roundtables/week11/day-2-components-timeline.md

```diff
@@ -0,0 +1,36 @@
+# Identifying Components and Timeline
+
+Today we are going to be discussing the Identifying Components Worksheet and the Timeline project in a round table fashion. For this discussion, think of your instructor as a mediator and guide, rather than as a lecturer.
+
+For this activity, you are encouraged to do the following:
+
+* Whenever possible, you should use voice.
+* You should pose questions to the group, not the instructor.
+* Similarly, you are encouraged to answer questions, with the instructor available to correct misconceptions and provide answers when no one else has an answer.
+* Say something, even if you think you might be wrong! (Nobody throws a frisbee well on their first try)
+* When discussing code, you are encouraged to use VS Code Liveshare, and or screen sharing through zoom.
+* Take notes!
+
+## Materials
+[Identifying Components Worksheet](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/React/exercises/identifying-components.md)
+
+[Timeline](https://github.com/Ada-C13/react-timeline)
+
+
+## Prompts
+You are encouraged to copy these questions into a document and take notes during the discussion.
+
+1. What elements/parts of the site felt like they should be components? Why did they feel like they should be components?
+    * If possible, everyone should share one
+2. Did you have any components that were repeated (as in, there were multiples of the same component on this page)? If yes, what parts of those components were similar with each other, and what parts were different from each other?
+    * If possible, everyone should share one
+3. What components were nested inside of another component? How did you determine that?
+    * If possible, everyone should share one
```

Comment:
I assume this is all referring specifically to the worksheet, right? Could you add something like "In the Identifying Components Worksheet..." to these so the context is more explicitly clear to student?

Reply:
Awesome insight, thank you! Will do